tests/integration/devfile/cmd_devfile_deploy_test.go

```diff
@@ -41,13 +41,13 @@ var _ = Describe("odo devfile deploy command tests", func() {
 		// from devfile
 		cmpName := "nodejs-prj1-api-abhz"
 		deploymentName := "my-component"
-
 		BeforeEach(func() {
 			helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
 			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile-deploy.yaml"), path.Join(commonVar.Context, "devfile.yaml"))
 		})
 		AfterEach(func() {
-			helper.Cmd("odo", "v2delete", "-af").ShouldPass()
+			// assuming we are inside the context directory since `odo deploy` cannot be run outside a context directory
+			helper.Cmd("odo", "delete", "component", "-f").ShouldPass()
```

Comment:
These delete are part of the cleanup, which is handled by deleting the namespace, I think it is not necessary and would be cleaner to not run the `odo delete` here (and in other places in the file)